docs/sources/dashboards/build-dashboards/create-dashboard/index.md

```diff
@@ -135,6 +135,9 @@ You can size a dashboard panel to suits your needs.
 1. To adjust the size of the panel, click and drag the lower-right corner of the panel.
 
 {{% docs/reference %}}
+[Dashboard]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/datasources#special-data-sources"
+[Dashboard]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/datasources#special-data-sources"
```

Comment:
I don't understand why this (and all the other ones) don't point to Cloud docs...
```suggestion
[Dashboard]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/datasources#special-data-sources"
```